packages/rollup-plugin-styles/src/runtime/inject-css.js

```
@@ -0,0 +1,74 @@
+    /** @type {HTMLStyleElement} */
+    let styleTag;
+
+    if (singleTag) {
+        let id = containers.indexOf(container);
+
+        if (id === -1) {
+            id = containers.push(container) - 1;
+            styleTags[id] = {};
+        }
+
+        styleTag = styleTags[id] && styleTags[id][position] ? styleTags[id][position] : (styleTags[id][position] = createStyleTag());
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)